go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/hashicorp/golang-lru v0.5.5-0.20210104140557-80c98217689d
 	github.com/holiman/bloomfilter/v2 v2.0.3 // indirect
 	github.com/karalabe/usb v0.0.2 // indirect
+	github.com/klauspost/pgzip v1.0.2-0.20170402124221-0bf5dcad4ada
```

Comment:
Any reason why v1.0.2 is used? Seems like [v1.2.6](https://github.com/klauspost/pgzip/releases/tag/v1.2.6) would be the latest version.